modules/ROOT/pages/deploy-resource-allocation.adoc

```diff
@@ -1,11 +1,11 @@
-= Resource Allocation on Anypoint Runtime Fabric
+= Resource Allocation and Performance on Anypoint Runtime Fabric
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
 Before deploying a Mule application to Runtime Fabric, determine the correct amount of resources to allocate. Determining resource allocation is also applicable when configuring the internal load balancers on Anypoint Runtime Fabric.
```

Comment:
Even though the product names are fully spelled in the titles, we should fully spell them in the first narrative mention:

    Before deploying...to Anypoint Runtime Fabric,...

"amount of resources" > "number of resources"

"...is also applicable when configuring" > "...is also important when you configure"?